packages/app-store/whereby/components/icon.tsx

```diff
@@ -0,0 +1,21 @@
+        <path d="m83.6263 5.97128c-1.46 0-2.859 1.37401-3.7208 3.22782l-1.4474-3.03842h-.1691c-.275 1.03538-1.567 1.62654-2.9868 1.60588v.16873c.8894.21121 1.2702 1.05605 1.8213 2.17641l3.7485 7.7125-.2543.4649c-.8054 1.5852-1.3553 2.1557-1.9271 2.1557-1.0378 0-1.123-1.5003-2.3724-1.5003-.741 0-1.3554.5074-1.3554 1.3947 0 .9929 1.1011 1.6265 2.2447 1.6265 1.6729 0 2.7107-1.3097 3.8336-3.4229l2.9016-5.4099c1.1644-2.1764 1.8006-3.1268 1.8006-4.75453.0012-1.30742-.7616-2.40709-2.117-2.40709zm-.1909 6.88952-.932 1.7964-2.3919-5.0219c.4579-.81155 1.2633-1.46584 2.4552-1.46584 1.1229 0 1.7154.69676 1.7154 1.83894.0012.7817-.2749 1.7528-.8467 2.8524z"></path>
+        <path d="m31.4821 15.2484v-6.52915c0-1.4578-.9112-2.43005-2.2665-2.43005-1.6303 0-3.6851 1.47961-4.7862 2.55631v-8.8110862h-.1484c-.6569 1.0353862-2.3505 1.6690062-3.982 1.6265362v.16874c.932.25368 1.2069.90912 1.2069 2.17637v11.24113c0 .9505-.4234 1.7115-1.1861 2.0283v.1687h5.3372v-.1687c-.7835-.4442-1.2287-1.0778-1.2287-1.9858v-5.8955c1.2494-.90912 2.2032-1.3522 2.9016-1.3522.7835 0 1.2288.57049 1.2288 1.73214v5.51556c0 .9091-.4027 1.5209-1.2288 1.9858v.1687h5.3373v-.1687c-.7617-.3157-1.1851-1.0767-1.1851-2.0271z"></path>
+        <path d="m71.1303 6.2892c-1.5244 0-2.9868 1.12033-3.791 1.98583v-8.2406062h-.1265c-.7628 1.0353862-2.5208 1.6690162-4.0879 1.6265362v.16874c.8687.25368 1.2702.92978 1.2702 2.17637v13.65053h.191l1.3979-1.5634c.6144.9723 1.7787 1.5634 3.1766 1.5634 3.516 0 5.9298-3.0005 5.9298-6.2754 0-3.00057-1.8845-5.092-3.9601-5.092zm-2.0548 10.7969c-1.0378 0-1.7362-.8448-1.7362-2.0076v-6.21232c.6984-.61297 1.4831-1.18346 2.3506-1.16165 1.483.02066 2.3298 1.69082 2.3505 4.24717.0218 2.8949-1.1643 5.1344-2.9649 5.1344z"></path>
+        <path d="m51.2641 6.28927c-1.1643 0-2.4356 1.2466-3.0282 2.43006v-2.55632h-.1691c-.7202 1.03538-2.4149 1.64835-4.0027 1.62654v.16874c.8687.25368 1.2921.95043 1.2921 2.17641v5.0288c0 .9722-.4234 1.7321-1.2702 2.1132v.1688h5.4638v-.1688c-.8686-.3799-1.2494-1.141-1.2494-2.1132v-5.91615c.3175-.40176.6776-.82418 1.101-.82418.7835 0 1.1011.95044 2.0756.95044.6776 0 1.3771-.54983 1.3771-1.47961-.0023-.92863-.7432-1.60473-1.59-1.60473z"></path>
+        <path d="m43.0884 9.28982c0-1.73214-1.7788-3.00054-4.0453-3.00054-3.4309 0-5.7607 2.95807-5.7607 6.06422 0 3.3805 2.5841 5.4099 5.1463 5.4099 2.0549 0 4.0453-1.2466 4.6597-3.4229l-.1485-.1263c-.6983 1.0354-1.9064 1.6059-3.1133 1.6059-1.5889 0-2.8809-.9091-3.4941-2.5357 3.2824-.5291 6.7559-1.4176 6.7559-3.99458zm-6.9043 3.48608c-.1266-.5073-.191-1.056-.191-1.6265 0-2.74688 1.1643-4.28963 2.6048-4.28963 1.1229 0 1.483.95044 1.483 1.96516 0 2.11207-1.9271 3.52857-3.8968 3.95097z"></path>
+      </g>
+    </svg>
+  );
+}
```

Comment:
why is this a function and not just a pure SVG? @zomars did you come up with this structure or @sean-brydon ?

Reply:
I think this is coming from @hariombalhara's CLI. But agreed, this should be a plain SVG and be served with our static endpoint.

Reply:
Yeah. It's from CLI. I used the latest app(zapier at that time) as the base. It had it.
Agreed that it should be static.

@sean-brydon. Please update `_baseApp` also when you fix it, so that all new apps don't have this problem.

Reply:
@CarinaWolli can you make a follow up PR and turn the function into static svg for zapier?

Reply:
@PeerRich yes, I will do that